filters/playartist.applescript

```diff
@@ -17,7 +17,22 @@ on getArtistResultListFeedback(query)
 		repeat with artistName in theArtists
 
 			set artistName to artistName as text
-			set theSong to (first track of playlist 2 whose artist is artistName)
+			-- Try to find a song with this artist name in any of the three fields
+			try
+				set theSong to (first track of playlist 2 whose artist is artistName)
+			on error
```

Comment:
The specific error code for `number -1728`. Can you please update this `on error` clause to specifically handle only that error? This ensures that any other errors are properly thrown (makes debugging easier).

```suggestion
			on error number -1728
```